### ISSUE TITLE
this interaction failed

### ISSUE THREAD

**Issue by EnzoCamilo** [hackermondev/discord-active-developer-badge#54]
whenever i attempt to run /ping, the interaction simply fails :/

**Comment by hackermondev** [comment]
@EnzoCamilo is your bot running? the repl on replit needs to be running when you run the command.

**Comment by EnzoCamilo** [comment]
it was running when i ran the command, idk why it didn't work

**Comment by Flaryanlxrd** [comment]
had same issue, i didnt changed something in code and replit running correctly, all steps are followed correctly, but any commands replies me "this interaction failed"

**Comment by EnzoCamilo** [comment]
Nope, not on mobileEm 16 de mai. de 2024, à(s) 11:18, NoahTH555 ***@***.***> escreveu:﻿
Are you on mobile? If so turn on Desktop mode and run
